src/ProjectTemplates/test/Templates.Tests/ApiTemplateTest.cs

```diff
@@ -38,6 +38,7 @@ public async Task ApiTemplateCSharp()
 
     [ConditionalFact]
     [SkipOnHelix("https://github.com/dotnet/aspnetcore/issues/47478", Queues = "OSX.13.Amd64.Open;Ubuntu.2004.Amd64.Open;Windows.11.Amd64.Client.Open;")]
```

Comment:
@JamesNK @mitchdenny I think this list has to be extended to include all [other queues where the test fail](https://dev.azure.com/dnceng-public/public/_build/results?buildId=261675&view=ms.vss-test-web.build-test-results-tab).

Ditto for other three tests.